2024-11-08 10:47:19.430249 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000284BDD9B4D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:48:19.441324 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000284BDD93610>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:49:19.462147 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000284BDE90050>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:50:19.499032 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000284BDD92E90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:51:26.228453 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDBB4D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:52:26.253344 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3610>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:53:26.268743 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0050>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:54:26.280218 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2E90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:55:26.292076 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2FD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:56:26.310047 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3750>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:57:26.327322 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2710>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:58:26.338581 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2C10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 10:59:26.350457 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2210>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:00:26.366212 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1810>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:01:26.378806 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1D10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:02:26.392911 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB02D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:03:26.404051 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0690>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:04:26.414993 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0A50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:05:26.430344 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0E10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:06:26.441497 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB11D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:07:26.453134 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1590>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:08:26.463773 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:09:26.476669 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1BD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:10:26.499094 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1A90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:11:26.517837 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2350>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:12:26.528839 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1F90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:13:26.540988 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB16D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:14:26.551996 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2D50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:15:26.563371 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3610>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:16:26.578061 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3110>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:17:26.597241 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1BD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:18:26.608290 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1810>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:19:26.618916 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1450>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:20:26.635870 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1090>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:21:26.649821 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0CD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:22:26.661617 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0910>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:23:26.684301 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0550>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:24:26.695935 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1D10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:25:26.750150 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:26:26.775843 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3250>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:27:26.789869 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3390>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:28:26.801372 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB34D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:29:26.815008 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:30:26.830879 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:31:26.842335 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:32:26.849997 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:33:26.865542 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1E50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:34:26.881112 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB2350>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:35:26.893992 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1F90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:36:26.904967 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0410>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:37:26.920711 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB07D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:38:26.931906 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0B90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:39:26.942870 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0F50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:40:26.953072 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1310>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:41:26.971015 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB16D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:42:26.984176 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1A90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:43:26.995981 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1D10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:44:27.007574 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1810>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:45:27.020169 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2210>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:46:27.036068 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2C10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:47:27.054424 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2710>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:48:27.066169 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3750>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:49:27.079958 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2FD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:50:27.096535 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2E90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:51:27.107899 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:52:27.119167 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1590>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:53:27.130022 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB11D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:54:27.142703 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0E10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:55:27.153781 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0A50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:56:27.170521 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0690>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:57:27.191153 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB02D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:58:27.210118 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1E50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 11:59:27.227848 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB2210>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:00:27.240694 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3110>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:01:27.251608 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3610>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:02:27.267623 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2D50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:03:27.278109 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB16D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:04:27.299369 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1F90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:05:27.316479 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2350>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:06:27.330677 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1A90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:07:27.342010 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1BD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:08:27.354080 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:09:27.366313 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1D10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:10:27.377325 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0550>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:11:27.396316 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0910>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:12:27.409282 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB0CD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:13:27.427426 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1090>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:14:27.444378 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1450>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:15:27.456627 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1810>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:16:27.467499 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEB1BD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:17:27.492945 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1E50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:18:27.510187 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:19:27.528075 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:20:27.550089 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:21:27.560764 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:22:27.572170 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB34D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:23:27.589002 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3390>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:24:27.602964 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3250>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:25:27.979511 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3890>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:26:27.996605 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3B10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:27:28.007579 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8050>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:28:28.022487 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8410>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:29:28.036106 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA87D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:30:28.048701 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8B90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:31:28.076387 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8F50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:32:28.091877 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9310>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:33:28.105219 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA96D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:34:28.118073 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9A90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:35:28.135547 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3B10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:36:28.146456 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3890>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:37:28.157785 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3250>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:38:28.175025 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3390>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:39:28.188477 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB34D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:40:28.202623 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:41:28.237892 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:42:28.250941 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:43:28.267673 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:44:28.280043 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1E50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:45:28.295543 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA96D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:46:28.310416 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9310>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:47:28.328167 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8F50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:48:28.340933 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8B90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:49:28.359450 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA87D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:50:28.382636 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8410>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:51:28.394772 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8050>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:52:28.409179 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1E50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:53:28.426980 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:54:28.437358 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:55:28.448738 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:56:28.464966 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:57:28.480433 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB34D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:58:28.491483 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3390>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 12:59:28.503284 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3250>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:00:28.517316 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3890>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:01:28.528245 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3B10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:02:28.544616 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8410>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:03:28.555731 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA87D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:04:28.566460 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8B90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:05:28.587554 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8F50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:06:28.599278 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9310>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:07:28.611666 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA96D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:08:28.624961 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9E50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:09:28.637899 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3B10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:10:28.649591 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3890>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:11:28.660480 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3250>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:12:28.674740 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3390>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:13:28.686337 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB34D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:14:28.699686 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:15:28.713105 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:16:28.724104 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:17:28.734952 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:18:28.749200 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1E50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:19:28.763437 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA96D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:20:28.786916 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9310>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:21:28.799046 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8F50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:22:28.815028 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8B90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:23:28.825414 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA87D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:24:28.838944 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8410>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:25:28.852044 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEAA210>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:26:28.864097 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1BD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:27:28.875429 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1A90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:28:28.893312 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2350>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:29:28.904162 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1F90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:30:28.915039 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB16D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:31:28.926576 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2D50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:32:28.950698 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3610>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:33:28.961347 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3110>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:34:28.972233 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB39D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:35:28.978640 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3D90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:36:28.993911 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8550>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:37:29.014149 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8910>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:38:29.029264 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8CD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:39:29.044490 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9090>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:40:29.057599 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9450>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:41:29.069538 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9810>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:42:29.086581 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEAA5D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:43:29.103694 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3B10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:44:29.115763 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3890>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:45:29.129194 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3250>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:46:29.142811 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3390>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:47:29.155069 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB34D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:48:29.170047 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:49:29.189917 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:50:29.201092 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:51:29.217210 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:52:29.232162 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEAA490>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:53:29.245168 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA96D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:54:29.256144 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9310>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:55:29.267086 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8F50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:56:29.299022 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8B90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:57:29.317495 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA87D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:58:29.335839 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8410>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 13:59:29.355350 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEAA990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:00:29.405017 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:01:29.419137 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:02:29.445781 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:03:29.465096 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:04:29.482020 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB34D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:05:29.496641 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3390>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:06:29.530093 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3250>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:07:29.548970 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3890>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:08:29.562150 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3B10>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:09:29.574306 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8410>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:10:29.592591 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA87D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:11:29.609959 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8B90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:12:29.637437 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8F50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:13:29.649618 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9310>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:14:29.666570 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA96D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:15:29.679480 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEAA490>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:16:29.709830 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEAAD50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:17:29.723884 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB39D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:18:29.742458 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3110>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:19:29.758941 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB3610>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:20:29.770753 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2D50>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:21:29.801418 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB16D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:22:29.820947 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1F90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:23:29.837805 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2350>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:24:29.854487 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1A90>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:25:29.866414 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1BD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:26:29.912092 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9810>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:27:29.941477 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9450>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:28:29.957169 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA9090>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 14:29:29.971081 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8CD0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:41:17.115727 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8910>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:42:17.706266 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA8550>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:43:17.723066 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEA82D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:44:17.745504 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDEAB110>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:45:17.762017 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB1950>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:46:17.773598 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2850>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:47:17.782648 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB20D0>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
2024-11-08 16:48:17.813418 - Status: DOWN - Code: HTTPSConnectionPool(host='www.wisecow.com', port=443): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x00000210EDDB2990>: Failed to resolve 'www.wisecow.com' ([Errno 11001] getaddrinfo failed)"))
